tests/acceptance/tests/basic_functionality/negative/misconfigure_lcm.rb

```diff
@@ -1,6 +1,6 @@
 require 'erb'
 require 'dsc_utils'
-test_name 'MODULES-2480 - C89505 - Misconfigure LCM and Attempt to Apply DSC Manifest'
+test_name 'MODULES-2480 - C89505 - Apply DSC Manifest with LCM refresh mode set to Push'
```

Comment:
@cowofevil I'm not sure how we should address this, since it's no longer a valid negative test.

We can delete the existing one and create a new one?  Or should I just do what I've done here, but move the file out of the `negative` directory, and rename to live alongside other tests?